docs/build-on-linea/tooling/data-indexers/goldsky/overview.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Overview
+sidebar_position: 1
+---
+
+# Goldsky Overview
+
+[Goldsky](https://goldsky.com) is a high-performance data indexing provider for Linea that makes it easy to extract, transform, and load on-chain data to power both application and analytics use cases. Goldsky offers two primary approaches to indexing and accessing blockchain data: Index (high-performance subgraphs) and Mirror (real-time data replication pipelines).
+
+## Index (high-performance subgraphs)
+
+[Index](goldsky-index) is a high-performance hosted subgraph solution with native support for Linea (both testnet and mainnet). Subgraphs allow you to define a data schema and how blockchain data should be indexed for querying, and can be deployed to Goldsky directly from the command line. If you’ve built with subgraphs in the past, you can work with Goldsky’s hosted service with no change to your development flow, and take advantage of several key benefits:
```

Comment:
I think our link checker doesn't understand `goldsky-index` and you'll need to modify this to `./goldsky-index`

https://github.com/Consensys/doc.zk-evm/actions/runs/5804284068/job/15747531741?pr=253